src/View/User/Includes/_welcome_card.html.twig

```diff
@@ -0,0 +1,13 @@
+{# User/Includes/_welcome_card.html.twig #}
+
+<div class="card" style="width: 18rem;">
```

Comment:
With the top comment

Reply:
we created 2 distinct elements at the start in our user stories